tsconfig.json

```diff
@@ -24,7 +24,8 @@
     "./optuna_dashboard/static/index.tsx"
   ],
   "include": [
-    "./optuna_dashboard/static/types/**/*"
+    "./optuna_dashboard/static/types/**/*",
+    "./tests/frontend_tests"
```

Comment:
Could you put `frontend_tests` directory on the repository root?

(Let me rename an existing `tests` directory with `python_tests` in the following pull requests.)

Reply:
No problem